pennylane/ops/qubit/qchem_ops.py

```diff
@@ -588,6 +617,16 @@ class OrbitalRotation(Operation):
         -1 / 2,
     ]
 
+    def parameter_frequencies(self):
+        return [
+            (
+                0.5,
+                1.0,
+                1.5,
+                2.0,
+            )
+        ]
```

Comment:
I think if you rewrite this (and above) as one line, black shouldn't complain?

Reply:
I think black _did_ this... :thinking: Will look into it.

Reply:
Ah, if one leaves a trailing coma although the tuple has more than one entry, black will do this for some reason :)